distributed/security.py

```diff
@@ -105,8 +105,11 @@ def _get_config_for_role(self, category, role, fields, per_role_fields):
 
     def _get_tls_context(self, tls, purpose):
         if tls.get('ca_file') and tls.get('cert'):
-            ctx = ssl.create_default_context(purpose=purpose,
-                                             cafile=tls['ca_file'])
+            try:
+                ctx = ssl.create_default_context(purpose=purpose,
+                                                 cafile=tls['ca_file'])
+            except AssertionError:
```

Comment:
I'm curious, where does the `AssertionError` come from? Did you mean `AttributeError`?